### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VoongChar**
Just having a play around

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # jubilant-palm-tree
 
 ## Helo
+
+Mike: - I have 3 brothers - One in London, Two in Melbourne.
+      - I once bought a van and drove around Europe for 3 months.
+      - I grew up in Christchurch.
```
Comment (VoongChar) [inline README.md:7]: Lie

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # jubilant-palm-tree
 
 ## Helo
+
+Mike: - I have 3 brothers - One in London, Two in Melbourne.
+      - I once bought a van and drove around Europe for 3 months.
```
Comment (VoongChar) [inline README.md:6]: tru

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
 # jubilant-palm-tree
 
 ## Helo
+
+Mike: - I have 3 brothers - One in London, Two in Melbourne.
```
Comment (VoongChar) [inline README.md:5]: tru

**Review by VoongChar — COMMENTED** [review]
@mjkenno

File: README.md (hunk 3 above)
Comment (VoongChar) [reply README.md:4]: @mjkenno

File: README.md (hunk 1 above)
Comment (mjkenno) [reply README.md:6]: Correct

File: README.md (hunk 2 above)
Comment (mjkenno) [reply README.md:5]: Correct

File: README.md (hunk 3 above)
Comment (mjkenno) [reply README.md:4]: Correct

**Pull request merged**